packages/data-models/src/language_model.js

```diff
@@ -420,12 +420,18 @@ class LanguageModel {
    * Returns alternate encodings for a word
    *
    * @param {object} params paramaters {}
-   *   @param {string} word the word
-   *   @param {string} preceding preceding word (optional)
-   *   @param {string} following following word (optional)
-   *   @param {string} encoding encoding name to filter the response to (optional)
-   *   @param {boolean} preserveCase if true will preserve the case (default is false)
-   *   @param {boolean} includeOriginal if true will include the original word even if it is unchanged (default is false)
+   * @param params.word
+   * @param {string} word the word
+   * @param {string} preceding preceding word (optional)
+   * @param {string} following following word (optional)
+   * @param {string} encoding encoding name to filter the response to (optional)
+   * @param {boolean} preserveCase if true will preserve the case (default is false)
+   * @param {boolean} includeOriginal if true will include the original word even if it is unchanged (default is false)
+   * @param params.preceding
+   * @param params.following
+   * @param params.encoding
+   * @param params.preserveCase
+   * @param params.includeOriginal
```

Comment:
I think the linter didn't understand our js docs here  because it looks like it repeated them

Reply:
Oh, I've missed that. I think the problem is that we declared parameters as independent variables and linter wanted them to be props of the `params` object. I will correct a JSDoc block.